src/client/dfuse/dfuse_main.c

```diff
@@ -261,6 +261,7 @@ show_help(char *name)
 	    "\n"
 	    "	-S --singlethread	Single threaded\n"
 	    "	-t --thread-count=count	Number of fuse threads to use\n"
+	    "   -e --eq-count=count     Number of event queues to use\n"
```

Comment:
is the eq-count required to match the thread-count?

Reply:
No, I will expand on this in the help option.

Reply:
This ended up being a can of worms, I've improved the text below but it's still not clear and --singlethread now isn't correct so I'll rework tomorrow but for now I wanted to re-push to get some testing overnight.

Reply:
im not sure users are going to understand how / when to use the non-default eq  count and how that affects the number of threads. it might help to provide some guidance on when the default is best (tc = cores, eq = 1 ?) or when something else is better.

Reply:
Agree, how these options work, are documented and what we recommend are stlll in-progress.